xwiki-platform-core/xwiki-platform-localization/pom.xml

```diff
@@ -33,10 +33,20 @@
   <description>XWiki Platform - Localization - Parent POM</description>
   <modules>
     <module>xwiki-platform-localization-api</module>
+    <module>xwiki-platform-localization-rest</module>
```

Comment:
This needs to be updated, right ?

Reply:
No because `xwiki-platform-localization-rest` contains `xwiki-platform-localization-rest-api` and `xwiki-platform-localization-rest-default`.
But https://github.com/xwiki/xwiki-platform/pull/1595/files#diff-e0609ee1c594bb59216372bd806f653443f22af8e1c91d94fe3b28fd30cda1b5R436 must be updated though

Reply:
done

Reply:
Indeed sorry, I did not notice that you put them one level below.